broker-daemon/block-order-worker/index.js

```diff
@@ -358,6 +358,34 @@ class BlockOrderWorker extends EventEmitter {
     return blockOrder
   }
 
+  /**
+   * Cancel all active orders for a given market
+   * @param  {string} market to cancel orders on
+   * @return {object} cancelledOrders and failedToCancelOrders as arrays of blockorderids
+   */
+  async cancelActiveOrders (market) {
+    this.logger.info('Cancelling all active orders for market', { market })
+
+    const blockOrders = await this.getBlockOrders(market)
+    const activeBlockOrders = blockOrders.filter(blockOrder => blockOrder.isActive)
+    const cancelledOrders = []
+    const failedToCancelOrders = []
+
+    await Promise.all(
+      activeBlockOrders.map(async (blockOrder) => {
+        try {
+          await this.cancelBlockOrder(blockOrder.id)
+          cancelledOrders.push(blockOrder.id)
+        } catch (e) {
+          this.logger.error('Failed to cancel order', e)
```

Comment:
probably need the blockorder id in the logger. Also - if you pass the error as the second parameter to the logger it doesn't work well, it's better like:

```
this.logger.error('Failed to cancel block order', { blockOrderId: blockOrder.id, error: e })
```